src/main/java/com/google/devtools/build/lib/bazel/rules/java/jdk.WORKSPACE

```diff
@@ -238,4 +238,11 @@ http_archive(
     ],
 )
 
+http_archive(
```

Comment:
If people want to specify their own rules_java, they will get a conflict here, right? Can you use maybe? example: https://source.bazel.build/bazel/+/master:src/main/java/com/google/devtools/build/lib/bazel/repository/local_config_platform.WORKSPACE;l=3?q=utils.bzl%5C%22,%20%5C%22maybe&ss=bazel